docker-compose.yaml

```diff
@@ -1,120 +1,93 @@
+    PIMCORE_INSTALL_MYSQL_PASSWORD: "pimcore"
+    PIMCORE_INSTALL_MYSQL_DATABASE: "pimcore"
+    PIMCORE_INSTALL_MYSQL_PORT: "3306"
+    PIMCORE_INSTALL_INSTALL_BUNDLES: "PimcoreApplicationLoggerBundle,PimcoreCustomReportsBundle,PimcoreGlossaryBundle,PimcoreSeoBundle,PimcoreSimpleBackendSearchBundle,PimcoreStaticRoutesBundle,PimcoreTinymceBundle,PimcoreUuidBundle,PimcoreWordExportBundle,PimcoreXliffBundle"
+  extra_hosts:
+    - "host.docker.internal:host-gateway"
 
 services:
   db:
     image: mariadb:10.5
-    container_name: coreshop-3-mariadb
-    networks:
-      - cors_dev
-      - coreshop_3
     working_dir: /application
-    command: [mysqld, --character-set-server=utf8mb4, --collation-server=utf8mb4_general_ci]
+    command: [ mysqld, --character-set-server=utf8mb4, --collation-server=utf8mb4_general_ci ]
     volumes:
-      - coreshop-3-database:/var/lib/mysql
+      - db:/var/lib/mysql
     environment:
       - MYSQL_ROOT_PASSWORD=ROOT
       - MYSQL_DATABASE=pimcore
       - MYSQL_USER=pimcore
       - MYSQL_PASSWORD=pimcore
-    ports:
-      - "3306:3306"
+    expose:
+      - "3306"
 
   nginx:
     image: nginx:stable-alpine
     volumes:
-      - .:/var/www/html:cached
-      -  ./.docker/nginx.conf:/etc/nginx/conf.d/default.conf:ro
+      - .:/var/www/html
+      - ./.docker/nginx.conf:/etc/nginx/conf.d/default.conf:ro
     depends_on:
       - php
-      - php-debug
-    networks:
-      - cors_dev
-      - coreshop_3
-    labels:
-      - traefik.enable=true
-      - traefik.http.routers.coreshop_3.rule=Host(`coreshop-3.localhost`)
-      - traefik.http.routers.coreshop_3.entrypoints=cors_dev
-      - traefik.http.routers.coreshop_3.tls=true
-      - traefik.http.services.coreshop_3.loadbalancer.server.port=80
-      - traefik.docker.network=cors_dev
-
-  nginx8:
-    image: nginx:stable-alpine
-    volumes:
-      - .:/var/www/html:cached
-      -  ./.docker/nginx8.conf:/etc/nginx/conf.d/default.conf:ro
-    depends_on:
-      - php
-      - php-debug
-    networks:
-      - cors_dev
-      - coreshop_3
-    labels:
-      - traefik.enable=true
-      - traefik.http.routers.coreshop_3_php8.rule=Host(`coreshop-3-php8.localhost`)
-      - traefik.http.routers.coreshop_3_php8.entrypoints=cors_dev
-      - traefik.http.routers.coreshop_3_php8.tls=true
-      - traefik.http.services.coreshop_3_php8.loadbalancer.server.port=80
-      - traefik.docker.network=cors_dev
+    ports:
+      - "80:80"
 
   php:
-    image: pimcore/pimcore:php8.1-latest
-    container_name: coreshop-php
-    networks:
-      - coreshop_3
-    environment:
-      COMPOSER_HOME: /var/www/html
-    depends_on:
-      - db
-    volumes:
-      - .:/var/www/html:cached
-
-  php-debug:
-    image: pimcore/pimcore:php8.1-debug-latest
-    container_name: coreshop-php-debug
-    networks:
-      - coreshop_3
+    <<: *php
+    image: coreshop/coreshop:PHP8.1-fpm-debug
+    build:
+      target: dev
+      args:
+        uid: ${USER_ID:-1000}
```

Comment:
ping @TanaseTeofil

Reply:
We've updated the guide to support Docker Desktop so that it can work on Windows and Mac as well.
We will provide native Linux docker setup later today or in the following days for people on Linux who do not wish to use Docker Desktop.